resources/flavors.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+      "is_automated": true,
+      "name": "bgruening/galaxy-stable",
+      "is_trusted": true,
+      "is_official": false,
+      "star_count": 8,
```

Comment:
The public dockerHub API doesn’t support jsonp at the moment. But when we have our own docker registry we can enable the jsonp feature and then this can be generated on the fly.

 

 

Van: Björn Grüning [mailto:notifications@github.com] 
Verzonden: 08 August 2015 16:42
Aan: galaxyFlavorsGenerators/galaxyFlavorGenerator <galaxyFlavorGenerator@noreply.github.com>
CC: Jeroen Galle <jeroen.galle@ugent.be>
Onderwerp: Re: [galaxyFlavorGenerator] added base image select (#16)

 

In resources/flavors.json <https://github.com/galaxyFlavorsGenerators/galaxyFlavorGenerator/pull/16#discussion_r36578875> :

> @@ -0,0 +1,114 @@
> +[
> +    {
> +      "is_automated": true,
> +      "name": "bgruening/galaxy-stable",
> +      "is_trusted": true,
> +      "is_official": false,
> +      "star_count": 8,

Is it possible to get such information on the fly from the DockerHub API or do we need to update this list?

—
Reply to this email directly or view it on GitHub <https://github.com/galaxyFlavorsGenerators/galaxyFlavorGenerator/pull/16/files#r36578875> .  <https://github.com/notifications/beacon/AL9cShyqF3KYWHZeaCrwtMQR4PzMs6f2ks5olgy9gaJpZM4FoBIl.gif>